pulsar-broker/src/main/java/com/yahoo/pulsar/broker/namespace/OwnedBundle.java

```diff
@@ -101,16 +101,29 @@ public void handleUnloadRequest(PulsarService pulsar) throws Exception {
             this.nsLock.writeLock().unlock();
         }
 
-        int unloadedTopics = 0;
+        int unloadedTopics = -1;
         try {
             LOG.info("Disabling ownership: {}", this.bundle);
-            pulsar.getNamespaceService().getOwnershipCache().disableOwnership(this.bundle);
-
-            // Handle unload of persistent topics
-            unloadedTopics = pulsar.getBrokerService().unloadServiceUnit(bundle).get();
-            pulsar.getNamespaceService().getOwnershipCache().removeOwnership(bundle);
+            pulsar.getNamespaceService().getOwnershipCache().updateBundleState(this.bundle, false);
+
+            // close topics forcefully
+            try {
+                unloadedTopics = pulsar.getBrokerService().unloadServiceUnit(bundle).get();
+            } catch (Exception e) {
+                // ignore topic-close failure to unload bundle
```

Comment:
while closing the topic we disconnects [producers, subscribers and replicator](https://github.com/rdhabalia/pulsar/blob/61c0b04f14c78365c76a96dc5e3413b7c44221d3/pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/persistent/PersistentTopic.java#L514) and if it is successful then we remove it from the [MLFactory](https://github.com/rdhabalia/pulsar/blob/61c0b04f14c78365c76a96dc5e3413b7c44221d3/managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java#L882) immediately. 
Disconnecting prod,sub,repl should be clean but if not for one of the topic then additionally we can schedule a retry only for unloading-bundle : if there was a failure while closing any topic and if we successfully remove the ownership-node later. 
any thought?